docs/tutorials/dataset_creation/observation_space_subseting.py

```diff
@@ -0,0 +1,108 @@
+# fmt: off
+"""
+Observation space subseting
+=========================================
+"""
+# %%%
+# In this tutorial you'll learn how to have :class:`minari.DataCollectorV0` only collect a subset 
+# of the observation space in PointMaze. Specifically, we'll be collecting observations using 
+# random actions on PointMaze_UMaze-v3 from `Gymnasium-Robotics <https://robotics.farama.org/envs/maze/point_maze/>`_
+# and omitting ``achieved_goal`` from the observation space of PointMaze.
+# This might be useful if you don't plan on training goal oriented learning agents 
+# on the Minari dataset generated.
+#
+# Please note that while only subseting the observation space of PointMaze is touched on in 
```

Comment:
I'd prefer if you avoid the verb subseting and use "..only a subset of the observation space ..."

Reply:
Done